project-p2-layouts/code/style.css

```diff
@@ -0,0 +1,177 @@
+/* Here in the css file you'll write your styling. make sure to link the css file to the html file. If it's linked correctly you should see a light blue background. */
+
+body {
+  background-color: white;
+}
+
+.top-header {
+  font-family: "Lucida Handwritting";
+  text-transform: uppercase;
+  font-size: 50pt;
+  text-align: center;
+  color: #023a02;
+}
+.menu {
+  font-family: "Arial";
+  text-transform: uppercase;
+  font-size: 12pt;
+  display: flex;
+  background-color: #00311a;
+  color: #eeeeee;
+  text-align: center;
+  padding: 20px;
+  cursor: pointer;
+  margin-top: 10px;
+  margin-bottom: 10px;
+  flex-direction: row;
+  justify-content: space-between;
+}
+a {
+  color: rgb(255, 255, 255);
+  text-decoration: none;
+}
+.welcome {
+  margin-top: 15pt;
+  font-family: "Arial";
+  position: relative;
+  text-align: center;
+  color: white;
+}
+.welcome-title {
+  font-size: 50pt;
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  overflow: hidden;
+  border-right: 0.15em solid rgb(255, 255, 255); /* The typwriter cursor */
+  white-space: nowrap; /* Keeps the content on a single line */
+  margin: 0 auto; /* Gives that scrolling effect as the typing happens */
+  letter-spacing: 0.15em; /* Adjust as needed */
+  animation: typing 3.5s steps(40, end), blink-caret 0.75s step-end infinite;
+}
+@keyframes typing {
+  from {
+    width: 0;
+  }
+  to {
+    width: 50%;
+  }
+}
+@keyframes blink-caret {
+  from,
+  to {
+    border-color: transparent;
+  }
+  50% {
+    border-color: transparent;
+  }
+}
+
+.footer {
+  font-family: "Arial";
+  text-transform: uppercase;
+  font-size: 12pt;
+  display: flex;
+  background-color: #00311a;
+  color: #eeeeee;
+  text-align: center;
+  padding: 20px;
+  cursor: pointer;
+  margin-top: 10px;
+  margin-bottom: 10px;
+  flex-direction: row;
+  justify-content: space-between;
+  scroll-behavior: smooth;
+}
+.aboutme {
```

Comment:
Small thing, but try to be consistent when naming classes (should be kebab-case)